### ISSUE TITLE
Rounded bordered input renders to underlined input

### ISSUE THREAD

**Issue by andidev** [GeekyAnts/NativeBase#641]
## react-native, react and native-base version
    "native-base": "2.0.12",
    "react": "15.4.1",
    "react-native": "0.42.0",
## Expected behaviour
Expected rounded borders of input
## Actual behaviour
The input is underlined instead of rounded borders
## Steps to reproduce
Use the example from the docs :
```xml
             <Container>
                <Content>
                    <Item rounded>
                        <Input placeholder='Rounded Textbox'/>
                    </Item>
                </Content>
            </Container>
```
## Screenshot of emulator/device
![image](https://cloud.githubusercontent.com/assets/571389/23877061/c6cacfd6-0840-11e7-8e32-129c0b2053e4.png)

## Is the bug present in both ios and android or in any one of them?
only tested ios

**Comment by andidev** [comment]
In fact a lot of the `Form` examples does not run and work as intended. I think the docs maybe is not updated?

E.g. the docs mentions `regular` attribute and `bordered` attribute. Which looks like the same. None of them seems to work though.

E.g. there are comments like `// Simple rectangular text input box` in the example that does not run.

**Comment by shivrajkumar** [comment]
Fixed in v2.1.3
